db/db.go

```diff
@@ -0,0 +1,51 @@
+// Package db implements main options of the game through a connection to database
+package db
+
```

Comment:
Another issue can detected by the following test case
`func TestDB_UserFund2(t *testing.T) {
	r := require.New(t)
	db := New()
	u := entities.User{
		Name:    "Jana",
		Balance: 300,
	}
	r.NoError(db.SaveUser(&u))


	for i := 0; i < 100; i++ {
		go func() {
			r.NoError(db.UserFund(1, 1))
		}()
	}
}`